src/components/Modal.js

```diff
@@ -190,12 +192,13 @@ const Modal = React.createClass({
           <div className='mx-modal-scrim' onClick={this.props.onRequestClose} style={Object.assign({}, styles.scrim, styles.overlay, this.props.isRelative && styles.relative)}></div>
           <div
             className='mx-modal-container'
-            ref={ref => this._modal = ref}
             style={Object.assign({}, styles.container, this.props.style)}
           >
             {this._renderTitleBar()}
             <div
+              aria-label={this.props.ariaLabel}
```

Comment:
hmm. we could potentially add some logic to do something like this if the title exists but in a lot of places internally we do not use a title.